student.library/src/main/java/tds/student/sql/data/OpportunityStatusExtensions.java

```diff
@@ -30,16 +28,6 @@ public static OpportunityStatusType parse (String statusString) {
     return parsedStatusType;
   }
 
-  public static OpportunityStatusType parseExamStatus(final String examStatusCode) {
```

Comment:
This requires exam-client to be a dependency which then creates a circular dependency.  Instead I moved the mapper code to student proper.